keyboards/download.php

```diff
@@ -81,11 +81,18 @@ function validateParameters(&$id, &$platform, &$mode, &$cid, &$target) {
    * Start a background PHP process to do the async work so we don't block the user
    */
   function triggerDownloadBackgroundProcesses($cid, $id, $platform, $mode) {
+    $bearer_token = getenv('TEAMCITY_TOKEN');
+    if($bearer_token === FALSE) {
+      error_log("ERROR: [download.php] TEAMCITY_TOKEN is not configured.");
```

Comment:
Do you want this in a block?
```
if(DEBUG) {
  error_log("ERROR: ...");
}
```

Reply:
This one is a configuration problem so I would like to be able to see it in the logs if it fails. The others may well be bad input so they just add noise to the log.